models/sdn-fabric-0.1.x/sdn-fabric.tree

```diff
@@ -0,0 +1,60 @@
+module: onf-switch
+  +--rw switch* [switch-id]
+     +--rw switch-id             inet:domain-name
+     +--rw display-name?         string
+     +--rw description?          string
+     +--rw attribute* [attribute-key]
+     |  +--rw attribute-key    yt:yang-identifier
+     |  +--rw value            string
+     +--rw model                 -> /sm:switch-model/switch-model-id
+     +--rw role                  enumeration
+     +--rw management
+     |  +--rw address?       inet:ipv4-address-no-zone
+     |  +--rw port-number?   inet:port-number
+     +--rw port* [cage-number channel-number]
+     |  +--rw cage-number           -> /sm:switch-model/port/cage-number
+     |  +--rw channel-number        -> /sm:switch-model/port/channel-number
+     |  +--rw display-name?         string
+     |  +--rw description?          string
+     |  +--rw speed?                identityref
+     |  +--rw dhcp-connect-point?   -> ../../dhcp-connect-point/dhcp-id
+     |  +--rw vlans
+     |     +--rw untagged?   -> ../../../vlan/vlan-id
+     |     +--rw tagged*     -> ../../../vlan/vlan-id
+     +--rw switch-pair
+     |  +--rw cage-number?      -> ../../port/cage-number
+     |  +--rw channel-number?   -> ../../port/channel-number
+     |  +--rw paired-switch?    -> /switch/switch-id
+     +--rw vlan* [vlan-id]
+     |  +--rw vlan-id         st:vlan
+     |  +--rw display-name?   string
+     |  +--rw description?    string
+     |  +--rw subnet          inet:ipv4-prefix
+     +--rw dhcp-connect-point* [dhcp-id]
```

Comment:
I was thinking dhcp-server would be a top-level object (same level as switch). The same server ought to be able to be attached to multiple switches ... but does it do so at potentially different addresses? Charles or Carmelo could clarify.

Reply:
I have moved this up to be a top level object

Reply:
Also I made the `connect-point` singular